fetch/peers/peers.go

```diff
@@ -1,11 +1,13 @@
 package peers
 
 import (
+	"slices"
```

Comment:
```suggestion
	"golang.org/x/exp/slices"
```

Reply:
But why?
All the funcs are present in the standard `slices` package as of Go 1.23.2

Reply:
Right, reading the documentation helps. I thought the `slices` package returns an iterator for the slice with the change, but actually both should behave the same. The linter complains that the return of `Sort` and `Compact` isn't used.

I also thought that they would do that in place 🤔 did the API change that much in recent Go versions or am I just misremembering things?